meet-data/nzpf/1214/entries.csv

```diff
@@ -9,4 +9,4 @@ Tania Pitt-Guthrie,Masters 1,F,63,100,70,125,295,Raw,SBD,1
 Amanda Young,Masters 1,F,84,105,77.5,140,322.5,Single-ply,SBD,1
 Bruce Park,Masters 5,M,74,115,70,170,355,Raw,SBD,1
 Steve Lousich,Masters 1,M,120+,300,200,230,730,Raw,SBD,1
-Steve Varga,Masters 1,M,110,300,210,260,770,Single-ply,SBD,1
+Steve Varga,Masters 1,M,120,300,210,260,770,Single-ply,SBD,1
```

Comment:
That seems like a big change! Why is that correct?

Reply:
The 110 class was removed and Steve Varga was competing at ~110kg for 2010-2008. He was also 120 for every meet after this.